presto-cache/src/test/java/com/facebook/presto/cache/TestCacheConfig.java

```diff
@@ -24,41 +23,41 @@
 import static com.facebook.airlift.configuration.testing.ConfigAssertions.assertFullMapping;
 import static com.facebook.airlift.configuration.testing.ConfigAssertions.assertRecordedDefaults;
 import static com.facebook.airlift.configuration.testing.ConfigAssertions.recordDefaults;
+import static com.facebook.presto.cache.CacheType.FILE_MERGE;
 import static io.airlift.units.DataSize.Unit.GIGABYTE;
 import static io.airlift.units.DataSize.Unit.MEGABYTE;
-import static java.util.concurrent.TimeUnit.DAYS;
 
 public class TestCacheConfig
 {
     @Test
     public void testDefaults()
     {
         assertRecordedDefaults(recordDefaults(CacheConfig.class)
+                .setCachingEnabled(false)
+                .setCacheType(null)
                 .setBaseDirectory(null)
                 .setValidationEnabled(false)
-                .setMaxInMemoryCacheSize(new DataSize(2, GIGABYTE))
```

Comment:
We should add `TestFileMergeCacheConfig` as well

Reply:
Added.